homeassistant/components/ws66i/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "ws66i",
+  "name": "Soundavo WS66i 6-Zone Amplifier",
+  "documentation": "https://www.home-assistant.io/integrations/ws66i",
+  "requirements": ["pyws66i==0.1"],
+  "codeowners": ["@ssaenger"],
+  "config_flow": true,
+  "iot_class": "local_polling"
```

Comment:
Can you discover this via DHCP ?

https://developers.home-assistant.io/docs/creating_integration_manifest?_highlight=dhcp#dhcp

Reply:
That is something I want to investigate as it seems like it can, but because I am new to HA and python, I am using the monoprice integration as a template for this one (since they are very similar), and that one can't do DHCP discovery. I want to get this approved and added before tackling DHCP.